### ISSUE TITLE
Migrate chart docs to docs.konghq.com

### ISSUE THREAD

**Issue by mflendrich** [Kong/charts#237]

**Comment by shaneutt** [comment]
After discussing with @mflendrich there's several considerations about the future of components we want to discuss as a team before we move forward on this. @Kong/team-k8s will need to have that conversation to unblock this.

**Comment by mflendrich** [comment]
I think that splitting this issue into 3 would unblock some of the work here:
1. retire the "Kong for Kubernetes" section of docs.konghq.com
2. migrate charts docs to docs.konghq.com
3. migrate operator docs to docs.konghq.com

I think that (1) and (2) are ready to be worked on. (3) is uncertain because implementation of service-apis will likely have substantial overlap with Kong Operator. Depending on the path forward for service-apis, the upcoming operator for `Gateway`s is likely to have significant feature overlap with Kong Operator.

**Comment by rainest** [comment]
Repurposing this issue for the chart doc migration, since it resides in the chart repo. The operator now has its own issue, and the follow-up task of de-duplicating content is https://github.com/Kong/team-k8s/issues/23

**Comment by rainest** [comment]
Base split layout proposed in the [docs/split branch](https://github.com/Kong/charts/tree/docs/split). It moves existing content under a [docs directory](https://github.com/Kong/charts/tree/docs/split/charts/kong/docs) and splits the README into several documents, with some changes in expectation of doc site stuff:

- ToC sections have been stripped, since the docs site generates them automatically.
- Titles are missing, since those will eventually use Jekyll titles
- Example configuration YAMLs left in the repo. Not sure it'd make sense to migrate them, but we'd need to consider tag/versioned links.
- The propose introduction.md is quite bare. It has a blurb that needs to be reworked (it describes "Kong for Kubernetes" rather than just the chart), repeated the install content (albeit with less info) and a Kong Nation link.
- CRD management still assumes that managed CRDs are part of the Kong chart, which doesn't really fit the Helm 3 model (if you want managed CRDs, use a separate chart). We probably should create a separate CRD chart, but existing docs still use the "turn off everything other than CRDs" system.

**Comment by rainest** [comment]
DOCS-1551 in internal Jira for coordination with the docs team.

**Comment by mflendrich** [comment]
Closing as "won't fix" for now due to limited demand for this change.

@scseanchow, please let us know if this should be bumped in importance.
